plugins/env/README.md

```diff
@@ -1,4 +1,43 @@
+## Env Plugin
 Plugin to manage ENV settings for projects and write .env files during deploy.
 
-Includes `/projects/:permalink/environment?deploy_group=permalink` endpoint that returns the .env content
+Includes `/projects/:permalink/environment?deploy_group=permalink` endpoint that returns the `.env` content
 for a project and deploy_group.
+
+## GitHub to manage environment variables
+This plugin has an option to use GitHub repository as source for environment variables. 
+The DEPLOYMENT_ENV_REPO must be set in samsons environment, for example https://github.com/foo/bar would be 
+`DEPLOYMENT_ENV_REPO=foo/bar`. Each project must opt-in to it via project settings.
+
+To enable the feature set the environment variable DEPLOYMENT_ENV_REPO to the organization 
+and repository for the deployment environment variables `.env`.  For an
+organization named `my_organization` and the repository named `deployment_environment_config` put this in Samson's 
+`.env` file:
+
+`DEPLOYMENT_ENV_REPO=my_organization/deployment_environment_config`
```

Comment:
delete too ... same as first

Reply:
I was describing what to do ant then provided an example of how to do it ...